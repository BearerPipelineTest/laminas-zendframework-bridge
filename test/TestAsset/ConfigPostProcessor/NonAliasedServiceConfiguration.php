<?php
return [
    'dependencies' => [
        'factories' => [
            'Zend\Form\Factory' => 'Some\Vendor\Zend\Form\Factory',
        ],
        'aliases' => [
            'foo' => 'Zend\Form\Factory',
        ],
    ],
];
