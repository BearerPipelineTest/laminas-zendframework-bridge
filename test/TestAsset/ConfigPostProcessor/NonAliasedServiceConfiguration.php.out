<?php
return [
    'dependencies' => [
        'factories' => [
            'Laminas\Form\Factory' => 'Some\Vendor\Zend\Form\Factory',
            'Laminas\Cache\Storage\StorageInterface' => 'Zend\ServiceManager\Factory\InvokableFactory',
        ],
        'aliases' => [
            'foo' => 'Laminas\Form\Factory',
            'Zend\Cache\Storage\StorageInterface' => 'Laminas\Cache\Storage\StorageInterface',
            'Zend\Form\Factory' => 'Laminas\Form\Factory',
        ],
    ],
];
